scalability/Makefile

```diff
@@ -209,7 +209,9 @@ clean: standard-clean
 		fi ; \
 	fi
 	# Generate line coverage information
-	${CURDIR}/../utils/scalabilitycov.sh "$*" "$$OUTPUT_DIR" ${SCALABILITY_COV_MAX_TIME}
+	if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \
```

Comment:
@rasoolmaghareh Lines 227-228 also has to be enclosed in the same if conditional so that line coverage field is not included in the spreadsheet in case coverage is disabled.

At Line 234, we should not have field "llvm-cov line coverage" in case coverage is disabled.

The way we generate small-coreutils.csv in Line 304 using cut needs change. cut needs to be given different field numbers in case coverage is disabled because we the field for line coverage is not there.